### PR TITLE
readme: Minor fixes for typo and link

### PR DESCRIPTION
**Pull request by imjoey**
* Use __OpenShift__ Python client
* Fix the wrong link to KubeVirt Python SDK

Signed-off-by: imjoey <majunjiev@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ $ export ANSIBLE_LIBRARY=<module_path>/lib/ansible/modules
 
 3. A working [Kubernetes configuration](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) is also required. It can also be created by issuing `oc login` if using KubeVirt with OpenShift.
 
-4. [Install KubeVirt Python SDK](https://github.com/openshift/openshift-restclient-python#installation)
```
Comment (machacekondra) [inline README.md]: Actually this should be:
```
4. [Install OpenShift Python SDK](https://github.com/kubevirt/client-python#installation--usage)
```

Comment (imjoey) [reply README.md]: Done. Thanks.

**Comment by pkliczewski** [comment]
Welcome and congrats for your first merge @imjoey :tada:

**Comment by imjoey** [comment]
😄  Thanks @pkliczewski .  🎉🎉🎉

**Pull request merged**